docs/source/user_guide/installation/index.rst

```diff
@@ -194,6 +194,37 @@ Dependency                                                  Minimum Version    p
 `tqdm <https://tqdm.github.io/>`__                          \-                 tqdm               For logging loops.
 =========================================================== ================== ================== ==================
 
+.. _install.jax_dependencies:
+
+Jax dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[jax]"``
+
+========================================================================= ================== ================== =======================
+Dependency                                                                Minimum Version    pip extra          Notes
+========================================================================= ================== ================== =======================
+`jax <https://jax.readthedocs.io/en/latest/notebooks/quickstart.html>`__  0.4.8              jax                For JAX solvers
+`jaxlib <https://pypi.org/project/jaxlib/>`__                             0.4.7              jax                Support library for JAX
+========================================================================= ================== ================== =======================
+
+.. _install.odes_dependencies:
+
+odes dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[odes]"``
+
+================================================================== ================== ================== =============================
+Dependency                                                         Minimum Version    pip extra          Notes
+================================================================== ================== ================== =============================
+`scikits.odes <https://scikits-odes.readthedocs.io/en/latest/>`__  \-                 odes               For scikits ODE & DAE solvers
+================================================================== ================== ================== =============================
+
+.. note::
+
+   Before installing `scikits.odes <https://scikits-odes.readthedocs.io/en/latest/>`_  make sure to install it's build-time requirements by running ``nox -s pybamm-requires``.
```

Comment:
I think it would be better to just link https://docs.pybamm.org/en/latest/source/user_guide/installation/GNU-linux.html#optional-scikits-odes-solver (`nox` is meant for developers and not users)